### PR TITLE
Add info on image automation

### PR DESCRIPTION
**Pull request by endakelly**
### Change description ###
Add a page with general info on image automation

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [x] Does this PR introduce a breaking change

### CODE REVIEW

File: source/aks/flux-image-automation.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+In the example above, only images tagged with something similar to `prod-abcd1234` will be applied.
+
+We have the image automation components set up on our PTL clusters only. 
+
+You can see more info about how the PTL clusters have been configured specifically in the guide on [flux patching](./patching-flux-example.html#prod-environments).
+
+To view the image repos and policies that are on the PTL clusters, you can run:
+
+```
+kubectl get imagerepository.image.toolkit.fluxcd.io -n flux-system
```
Comment (thomast1906) [inline source/aks/flux-image-automation.html.md.erb:57]: Want to add examples of the output? May be handy to reference

Comment (endakelly) [reply source/aks/flux-image-automation.html.md.erb:57]: Updated

Comment (endakelly) [reply source/aks/flux-image-automation.html.md.erb:57]: @thomast1906 might reviewing this again? I completely forgot this was open as well

**Review by thomast1906 — APPROVED** [review]
Is it worth adding to a aks/flux/* folder? or aks/troubleshooting/flux/* as I feel this will grow

**Pull request merged**